data/sql/updates/db_world/0000_00_00_00.sql

```diff
@@ -0,0 +1,452 @@
+UPDATE `creature_template` SET `ScriptName` = 'npc_og_assistants' WHERE `entry` IN (39273, 39910);
+UPDATE `creature_template` SET `ScriptName` = 'npc_og_i_tank' WHERE `entry` = 39819;
+UPDATE `creature_template` SET `AIName` = 'AggresorAI' WHERE `entry` IN (39755, 39836);
+UPDATE `creature_template` SET `VehicleId` = 0, `ScriptName` = 'npc_og_camera_vehicle' WHERE `entry` = 40479;
+UPDATE `creature_template` SET `mechanic_immune_mask` = 8192 WHERE `entry` IN (39860, 39826, 39799, 39819, 39273, 39910, 39837);
+
+DELETE FROM `creature` WHERE `id` IN (39273, 39910);
+INSERT INTO `creature` (`guid`,`id`,`map`,`spawnMask`,`phaseMask`,`modelid`,`equipment_id`,`position_x`,`position_y`,`position_z`,`orientation`,`spawntimesecs`,`spawndist`,`currentwaypoint`,`curhealth`,`curmana`,`MovementType`,`npcflag`,`unit_flags`,`dynamicflags`) VALUES
+(250249, 39273, 0, 1, 256, 0, 0, -5423.01, 535.254, 386.516, 5.23555, 300, 0, 0, 630000, 0, 0, 0, 134217728, 0),
+(250248, 39910, 0, 1, 256, 0, 0, -5427.93, 532.323, 386.85, 5.27046, 300, 0, 0, 630000, 0, 0, 0, 0, 0);
+UPDATE `creature` SET `position_x` = -5424.462891, `position_y` = 531.410095, `position_z` = 386.743347, `orientation` = 5.2 WHERE `id` = 39271;
+UPDATE `creature` SET `phaseMask`= 1 WHERE `id` = 7937;
+
+DELETE FROM `vehicle_template_accessory` WHERE `entry` = 39860;
+
+DELETE FROM `creature_template_addon` WHERE `entry` IN (39820, 39273, 39910);
+INSERT INTO `creature_template_addon` (`entry`, `path_id`, `mount`, `bytes1`, `bytes2`, `emote`, `auras`) VALUES
+(39820, 0, 0, 0, 0, 0, 74311),
+(39273, 0, 9473, 0, 0, 0, ''),
+(39910, 0, 6569, 0, 0, 0, '');
+
+UPDATE `creature_template` SET `modelid2` = 0 WHERE `entry` = 39903;
+UPDATE `creature_template` SET `speed_run` = 1.25, `faction` = 1771 WHERE `entry` = 39273;
+UPDATE `creature_template` SET `speed_run` = 1.29, `faction` = 1771 WHERE `entry` = 39910;
+
+DELETE FROM `spell_area` WHERE (`spell` = 74310) AND (`area` IN (1, 135, 721));
+INSERT INTO `spell_area` VALUES
+(74310, 721, 25287, 25393, 0, 0, 2, 1, 74, 11),
+(74310, 135, 25287, 25393, 0, 0, 2, 1, 74, 11);
+
+UPDATE `gameobject` SET `phaseMask` = 257 WHERE `id` = 194498;
+UPDATE `gameobject` SET `phaseMask` = 256 WHERE `id` = 202922;
+UPDATE `gameobject` SET `spawntimesecs` = -1 WHERE `guid` NOT IN (2482, 2469, 2461, 2458, 2454, 2453, 2466, 2475) and `id` = 194498;
+
+DELETE FROM `spell_scripts` where `id` IN (74412, 75510) AND `command` = 6;
+INSERT INTO `spell_scripts` (`id`, `command`, `x`, `y`, `z`, `o`) VALUES
+(74412, 6, -4842.156738, -1277.52771, 501.868256, 0.84),
+(75510, 6, -5164.767578, 556.341125, 423.753784, 25.29);
+
+DELETE FROM `spell_dbc` WHERE `id` = 75517;
+INSERT INTO `spell_dbc` VALUES (75517,0,0,384,1024,4,0,0,8,0,0,0,0,0,1,0,0,0,0,0,0,0,245,1,0,-1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,8,'Tumultuous Earthstorm');
```

Comment:
The culprit for the build failing

Reply:
btw need to add the column names in the sql